azurecontainerapps.ts

```diff
@@ -460,6 +483,13 @@ export class azurecontainerapps {
             environmentVariables.push(`ORYX_RUNTIME_PORT=${this.targetPort}`);
         }
 
+        // Set the user defined environment variables that should be propagated to the builder
+        if (!this.util.isNullOrEmpty(this.buildEnvironmentVariables)) {
+            this.buildEnvironmentVariables.match(/"[^"]*"|\S+/g).forEach((envVar) => {
```

Comment:
Let's set `/"[^"]*"|\S+/g` as a global RegEx property that we can reference instead of the direct RegEx just in case we need to make any changes to it in the future.